### ISSUE TITLE
Send bitcoin networks - multiple transaction outputs

### ISSUE THREAD

**Issue by PeKne** [trezor/trezor-suite#12944]
Allow to create multiple outputs within a single transaction according to [Figma spec](https://www.figma.com/design/g9LLS8q9IpUY8UTTBvQAJF/Send--Final-?node-id=2859-9366&t=Ti160nOPf1zwIY7d-4).

<img width="2076" height="2858" alt="Image" src="https://github.com/user-attachments/assets/0c086b4c-2aaa-4571-8adf-504e73201721" />

- [ ] `SendRecipientsCountChanged` analytics event is dispatched when is recipient added/removed

**Comment by tomasklim** [comment]
~@PeKne please check and add info~

Waiting for @hynek-jina to say if it still makes sense to implement multi recipient bitcoin send + product specifications for creating design and development

**Comment by hynek-jina** [comment]
@tomasklim Yes, It makes sense to have multiply recipients in bitcoin send.

There is already [some design for it](https://www.figma.com/design/g9LLS8q9IpUY8UTTBvQAJF/Send--Final-?node-id=2725-5853&t=7ZAqubUr35KdA3LH-4). 

<img width="390" height="848" alt="Image" src="https://github.com/user-attachments/assets/38ed081c-0b0b-4f3d-b5a7-3091ac092ec0" />

I assume that click on `+ Add recipient` would add recipient and amount section, but please double-check with @shenkys 
Try to keep aligned logic with desktop. For example the recipient naming and ordering, buttons order, button icons, ...

**Comment by matous-jemelka** [comment]
The original link to design didn't work for me, so I have recreated the design to match the desktop.
